frontend/src/metabase/lib/query/normalize.js

```diff
@@ -0,0 +1 @@
+export { normalize } from "cljs/metabase.mbql.js";
```

Comment:
Moved `cljs` import to a JS file as TypeScript can't resolve it when doing a type check